modules/apps/captcha/captcha-taglib/src/main/java/com/liferay/captcha/taglib/servlet/taglib/CaptchaTag.java

```diff
@@ -54,18 +59,38 @@ protected void setAttributes(HttpServletRequest httpServletRequest) {
 			"liferay-captcha:captcha:url", _getURL(httpServletRequest));
 	}
 
+	private String _getPortletId(HttpServletRequest httpServletRequest) {
+		return PortalUtil.getPortletId(httpServletRequest);
+	}
+
```

Comment:
I think there is no need to create a separate function here.

Reply:
Fixed it. Thanks